solr/bin/solr

```diff
@@ -437,9 +437,9 @@ function print_usage() {
     echo ""
     echo "  -p <port>     Specify the port the Solr HTTP listener is bound to"
     echo ""
-    echo "  -all          Find and stop all running Solr servers on this host"
+    echo "  --all         Find and stop all running Solr servers on this host"
```

Comment:
In hindsight this seems low-priority IMO.  Other CLI tools aren't necessarily consistent in what opt-variants they offer from param to param.  But I created a JIRA ticket in case it strikes someone as important - SOLR-17345.